gaphor/ui/toolbox.py

```diff
@@ -109,7 +109,11 @@ def on_collapsed(widget, prop, index):
             self.properties.set("toolbox-collapsed", collapsed)
 
         for index, (title, items) in enumerate(toolbox_actions):
-            tool_item_group = Gtk.ToolItemGroup.new(title)
+            tool_item_group = Gtk.ToolItemGroup.new("")
+            label = Gtk.Label.new(title)
+            label.set_halign(Gtk.Align.START)
+            label.show()
+            tool_item_group.set_label_widget(label)
```

Comment:
Probably could've used `tool_item_group.get_label_widget().set_halign(..)` too, but this is fine.

Reply:
@amolenaar Thanks for showing me that! That is a much nicer implementation and I pushed 096f7fb2 with this.